knowledge-base/codacy/codacy-analysis-cli-action/action-security.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Codacy Analysis CLI'
+outbound-endpoints:
+  - fqdn: raw.githubusercontent.com
+    port: 443
+    reason: to fetch source code
+  - fqdn: '*.docker.io'
+    port: 443
+    reason: to access docker 
+  - fqdn: auth.docker.io
+    port: 443
+    reason: to authenticate docker credentials
+  - fqdn: production.cloudflare.docker.com
+    port: 443
+    reason: to fetch content
```

Comment:
Please change `to fetch content` to `to pull image configuration`
Got the reason from here: https://www.reddit.com/r/docker/comments/gi9s3x/dns_error_when_pulling_docker_images/